apps/prairielearn/src/pages/shared/assessmentStatDescriptions.js

```diff
@@ -4,6 +4,13 @@ var stat_descriptions = {
     non_html_title: 'Mean',
     description: 'Mean score of a question is the average score for all students on the question.',
   },
+  // adding median score 
+  MEDIAN_SCORE: {
+    title: 'Median',
+    non_html_title: 'Median',
+    description:
+      'Median score of a question is the score which separates the lower half and the upper half of students scores.',
```

Comment:
We can bikeshed the best description here; here's a proposal:

```suggestion
      'Median score of a question is the 50th percentile score for all students on the question.',
```